server/mailet/mailets/src/main/java/org/apache/james/transport/mailets/delivery/MailDispatcher.java

```diff
@@ -198,11 +199,15 @@ private Mono<Void> storeMailWithRetry(Mail mail, MailAddress recipient) {
         AtomicInteger remainRetries = new AtomicInteger(retries.orElse(0));
 
         Mono<Void> operation = Mono.from(mailStore.storeMail(recipient, mail))
-            .doOnError(error -> LOGGER.warn("Error While storing mail. This error will be retried for {} more times.", remainRetries.getAndDecrement(), error));
+            .doOnError(OverQuotaException.class, e -> LOGGER.info("Could not store mail due to quota error", e))
```

Comment:
```suggestion
            .doOnError(OverQuotaException.class, e -> LOGGER.info("Could not store mail due to quota error fur user {}", user))
```

 -> let's have the user it is functionally important
 
 I am not convinced of the need log th full stacktrace in an info log.